scripts/stablex/get_auction_elements.js

```diff
@@ -14,7 +14,7 @@ const argv = require("yargs")
   })
   .version(false).argv
 
-const { decodeOrdersBN } = require("../../src/encoding")
+const { getOrdersViaPaginatedApproach } = require("./utilities")
```

Comment:
nit: I think this naming is a bit unnecessarily verbose. Maybe just `getOrdersPaginated`? Also why do we put it in utilities when it's only used once? Are you planning to move it somewhere else as well?

Reply:
Yeah, I am using it in the owl-liquidity provision script as well.